### ISSUE TITLE
Add Hubot Sans

### ISSUE THREAD

**Issue by RosaWagner** [google/fonts#5541]
**Font Project Git Repo URL:**
https://github.com/github/hubot-sans

**Comment by chrissimpkins** [comment]
Still blocked on the RFN issue.  Updated issue metadata accordingly and assigned Dave.

**Comment by chrissimpkins** [comment]
We resolved the RFN issue with the GitHub team and are prepared to continue with the QA.  Do we need anything else from type design team @RosaWagner?

**Comment by chrissimpkins** [comment]
Upstream is working on design changes that will cause visible H spacing regressions for users. We are going to wait for these to land before we onboard this family.

Tentatively pushing back to Q4 2023

**Comment by chrissimpkins** [comment]
Still waiting on information from the GitHub team on the timing of their font development updates / release

**Comment by RosaWagner** [comment]
status: no answer from the github team about the updated files yet. When they give an update, the onboarding process will have to start over as usual. All the PRs I made were merged, so there is nothing to do until they deliver new files.

**Comment by chrissimpkins** [comment]
The GitHub team verified this week that the development is nearly complete.  Will add this project to the Q1 queue.

**Comment by lisaschelch** [comment]
Is this project canceled or on hold? If Hubot does not get into Google Fonts, we need to replace it soon.

**Comment by yanone** [comment]
> Is this project canceled or on hold? 

On hold until sources are ready.

> If Hubot does not get into Google Fonts, we 

Who is “we”?

> need to replace it soon.

With what?

Sorry, I don't understand.

**Comment by chrissimpkins** [comment]
Github team told us that the onboarding of this project will be pushed back to ~ Q3 2024. They need more time on this one.

**Comment by emmamarichal** [comment]
@chrissimpkins Status: PR upstream opened here: https://github.com/github/hubot-sans/pull/37
